Dockerfile.konflux

```diff
@@ -0,0 +1,72 @@
+FROM quay.io/bcook/bazel:ubi8-bazel-5 as builder
+
+USER root
+
+RUN dnf update -y -q && \
+  dnf install -y -q \
+  which \
+  patch \
+  gcc \
+  clang \
+  cmake \
+  make \
+  openssl \
+  ca-certificates \
+  unzip \
+  git \
+  findutils \
+  python3
+
+COPY . /mlmd-src
+WORKDIR /mlmd-src
+
+# Running in offline mode with --nofetch arg, cache and deps must be cloned 
+# into the local root bazel cache
+# "-std=c++17" is needed in order to build with ZetaSQL.
+RUN bazel build -c opt --action_env=PATH \
+  --define=grpc_no_ares=true \
+  //ml_metadata/metadata_store:metadata_store_server \
+  --cxxopt="-std=c++17" --host_cxxopt="-std=c++17"
```

Comment:
guess no harm to keep -`-verbose_failures`

Reply:
Done